GitVersionConfig.yaml

```diff
@@ -3,3 +3,5 @@ next-version: 6.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```

Comment:
Also, v4 is well underway. It fixes a bunch of commit counting issues and it changes the default tags from unstable to alpha but bumps things in a different way.